pyro/contrib/funsor/handlers/__init__.py

```diff
@@ -8,13 +8,18 @@
     mask, reparam, scale, seed, uncondition,
 )
 
-
+from .enum_messenger import EnumMessenger, queue  # noqa: F401
```

Comment:
note: one way to avoid the F401 errors is to use the imported names in `__all__`; I am ambivalent between `noqa` comments and `__all__`.